### PR TITLE
Display Extension Preferences app

### PR DESCRIPTION
**Pull request by iaguis**
Also, change the name so it's not too long and use a reasonable icon.

https://phabricator.endlessm.com/T14161

### CODE REVIEW

File: data/eos-shell-extension-prefs.desktop.in.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-_Name=GNOME Shell Extension Preferences
+_Name=Extensions
 _Comment=Configure GNOME Shell Extensions
```
Comment (aperezdc) [inline data/eos-shell-extension-prefs.desktop.in.in]: Maybe we want the comment changed as well. For example to “Configure desktop extensions” if the reference to GNOME is to be removed.

File: data/eos-shell-extension-prefs.desktop.in.in [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-_Name=GNOME Shell Extension Preferences
+_Name=Extensions
```
Comment (aperezdc) [inline data/eos-shell-extension-prefs.desktop.in.in]: Seeing an icon that says just “Extensions”, they will wonder what the extensions are for. Maybe we should consider using ”Desktop extensions” instead, dunno.

File: data/eos-shell-extension-prefs.desktop.in.in (hunk 1 above)
Comment (iaguis) [inline data/eos-shell-extension-prefs.desktop.in.in]: I guess we'll need to change the app title too then

Comment (iaguis) [inline data/eos-shell-extension-prefs.desktop.in.in]: Here I mean https://github.com/endlessm/eos-desktop/blob/05540e732bd5e67a160709f1af60d4d2433d84cb/js/extensionPrefs/main.js#L148

File: data/eos-shell-extension-prefs.desktop.in.in (hunk 2 above)
Comment (iaguis) [inline data/eos-shell-extension-prefs.desktop.in.in]: It's annoying that it's wrapped:

image

**Pull request merged**